addons/project_hr_expense/models/project.py

```diff
@@ -46,13 +46,14 @@ def action_open_project_expenses(self):
 
     def _get_stat_buttons(self):
         buttons = super(Project, self)._get_stat_buttons()
-        buttons.append({
-            'icon': 'money',
-            'text': _('Expenses'),
-            'number': self.expenses_count,
-            'action_type': 'object',
-            'action': 'action_open_project_expenses',
-            'show': self.user_has_groups('hr_expense.group_hr_expense_team_approver') and self.expenses_count > 0,
-            'sequence': 10,
-        })
+        if self.user_has_groups('hr_expense.group_hr_expense_team_approver'):
+            buttons.append({
+                'icon': 'money',
+                'text': _('Expenses'),
+                'number': self.expenses_count,
+                'action_type': 'object',
+                'action': 'action_open_project_expenses',
+                'show': self.user_has_groups('hr_expense.group_hr_expense_team_approver') and self.expenses_count > 0,
```

Comment:
Since we are sure the user has the group, you can avoid to check again if the user has the group.
```suggestion
                'show': self.expenses_count > 0,
```